src/reconcile/rec_row.c

```diff
@@ -399,13 +402,13 @@ __wt_rec_row_int(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page)
         page_del = NULL;
         if (__wt_off_page(page, addr)) {
             page_del = cms.state == WT_CHILD_PROXY ? &cms.del : NULL;
-            __wt_rec_cell_build_addr(session, r, addr, NULL, WT_RECNO_OOB, page_del);
+            WT_ERR(__wt_rec_cell_build_addr(session, r, addr, NULL, WT_RECNO_OOB, page_del, NULL));
```

Comment:
I think we cannot pass NULL here. We need to first try to unpack the previous ps if it is writtin.